CoreScriptsRoot/Libraries/RbxUtility.lua

```diff
@@ -927,14 +927,21 @@ local function Create_PrivImpl(objectType)
 
 		--make the object to mutate
 		local obj = Instance.new(objectType)
+		local parent = nil
 
 		--stored constructor function to be called after other initialization
 		local ctor = nil
 
 		for k, v in pairs(dat) do
 			--add property
 			if type(k) == 'string' then
-				obj[k] = v
+				if k == 'Parent' then
```

Comment:
Should we FFlag this change? While it seems to be better all around there could be some unintended consequences.